src/main/scala/com/github/tkawachi/doctest/DoctestPlugin.scala

```diff
@@ -85,10 +69,10 @@ object DoctestPlugin extends AutoPlugin {
    */
   val doctestGenSettings = Seq(
     doctestTestFramework := (doctestTestFramework ?? ScalaCheck).value,
-    doctestWithDependencies := (doctestWithDependencies ?? true).value,
     doctestDecodeHtmlEntities := (doctestDecodeHtmlEntities ?? false).value,
     doctestMarkdownEnabled := (doctestMarkdownEnabled ?? false).value,
     doctestMarkdownPathFinder := baseDirectory.value * "*.md",
+    testFrameworks            += new TestFramework("utest.runner.Framework"),
```

Comment:
SBT suports natively ScalaTest, ScalaCheck and Specs2, if I'm not mistaken.
We are just adding an additional test framework to the list SBT supports natively.
The ability to instantiate a given test framework depends on the fact of finding it in the CLASSPATH, or not!